src/imp/openssl.rs

```diff
@@ -54,7 +55,8 @@ fn supported_protocols(
         | SslOptions::NO_SSLV3
         | SslOptions::NO_TLSV1
         | SslOptions::NO_TLSV1_1
-        | SslOptions::NO_TLSV1_2;
+        | SslOptions::NO_TLSV1_2
+        | SslOptions::NO_TLSV1_3;
```

Comment:
OpenSSL versions too old for set_min_proto_version won't have TLSV1_3 support afaik.

Reply:
I'm not sure what you mean. The OpenSSL versions that don't have min protocol config are very old and no longer supported I think (< v1.1.0), while TLS1.3 was introduced at OpenSSL v1.1.1, also pretty old by now.

Reply:
This code is only used for old versions of OpenSSL that won't have a `NO_TLSV1_3` symbol.

Reply:
I see. Didn't notice this is inside a `#[cfg()]` block.